IFC4x3/Properties/p/PowerCapacity_1mAfVHVQfDtBcb3$W2OBiX/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PowerCapacity_1mAfVHVQfDtBcb3W2OBiX" Name="PowerCapacity" UniqueId="702a97d1-7daa-4ddc-b9a5-0ff80260bb21" PrimaryDataType="IfcPowerMeasure" />
```

Comment:
Is this similar to NominalPower?

Reply:
Power capacity is how much energy is stored in the battery
NominalPower consist of the maximal power allowed to flow through the equipment

Reply:
Then the wrong unit is being used capacity is customarily measured in mAh (milliAmp hours) for batteries this means you should use `IfcElectricChargeMeasure`, where the units are specified on the property (default for charge is coulombs).

if you need pure power capacity Wh (Watt hours) then this is equivelent to Joules and therefore is `IfcEnergyMeasure`, as before unit should be specified because the default is Joules.